src/SlackHookHandler.ts

```diff
@@ -85,7 +85,7 @@ export class SlackHookHandler extends BaseSlackHandler {
         }
     }
 
-    private onRequest(req: IncomingMessage, res: ServerResponse) {
+    public _onRequest(req: IncomingMessage, res: ServerResponse) {
```

Comment:
This seems quite cheeky. If you're mocking stuff, why not `hookHandler["onRequest"]` to get around the public/private check.